### PR TITLE
Update team docs to include our usage of the occupy hand signals so n…

### PR DESCRIPTION
**Pull request by Rory80Hz**
…ew team members can understand them

### CODE REVIEW

File: docs/team/hand_signals.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+We have found these aid in communication and give everyone a voice in larger team discussions.
+
+## The Signals
+![Occupy Hand Signals](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b1/OccupyHandSignals.pdf/page1-440px-OccupyHandSignals.pdf.jpg)
+
+> [Occupy Hand Signals](https://en.wikipedia.org/wiki/File:OccupyHandSignals.pdf) by Ruben de Haas is licensed under [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/deed.en)
+
+### Disclaimer
+This is not an official endorsement of the Occupy movement itself.
```
Comment (combor) [inline docs/team/hand_signals.md]: Is it non-offcial endorsement then?

Not sure if this line necessary as we clearly state "Occupy Hand Signals" not "Occupy movement support"

Comment (Rory80Hz) [inline docs/team/hand_signals.md]: Spoke to our comms team about this and they asked that we include some sort of disclaimer about this! Maybe the word official is not needed here.

File: docs/team/hand_signals.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Occupy Hand Signals
```
Comment (saliceti) [inline docs/team/hand_signals.md]: This line can be removed as the title is already at the top, and it causes repetition on the side bar.

File: docs/team/hand_signals.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+It took a little getting used to, but after reviewing at a number of retrospectives as a team we have adopted the use of the [Occupy hand signals](https://en.wikipedia.org/wiki/Occupy_movement_hand_signals). 
+
+We have found these aid in communication and give everyone a voice in larger team discussions.
+
+## The Signals
+![Occupy Hand Signals](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b1/OccupyHandSignals.pdf/page1-440px-OccupyHandSignals.pdf.jpg)
+
+> [Occupy Hand Signals](https://en.wikipedia.org/wiki/File:OccupyHandSignals.pdf) by Ruben de Haas is licensed under [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/deed.en)
+
+### Disclaimer
```
Comment (saliceti) [inline docs/team/hand_signals.md]: Using the same indentation as `## The Signals` above would look better

**Pull request merged**